_posts/2023/2023-02-13-try-not-to-loop-through-associations-inside-jobs.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Try not to loop through associations inside jobs"
+description: "Your local data isn’t the same as in production"
+layout: article
+category: ruby
+image:
+  base: '2022/try-not-to-loop-through-associations-inside-jobs'
+  alt: "Roller coaster loops"
+  credit: "Priscilla Du Preez"
+  source: "https://unsplash.com/photos/FOsina4f7qM"
+
+---
+
+One of the downside of using an level of abstraction above your database is that it’s easy to accidentally give yourself performance headaches via running multiple very similar queries.
```

Comment:
```suggestion
One of the downsides of using Ruby to interact with your database rather than SQL directly is that it’s easy to accidentally give yourself performance headaches via running multiple very similar queries. Although the advantages of using Active Record (or something similar) are generally worth it!
```